pkg/ocm/helpers_test.go

```diff
@@ -453,3 +453,19 @@ var _ = Describe("ClusterDomainPrefixValidator()", func() {
 		Entry("should error when domain prefix is longer than 15 chars", strings.Repeat("h", 16), true),
 		Entry("should not error when domain prefix valid", strings.Repeat("h", 15), false))
 })
+
+var _ = Describe("ValidateClaimValidationRules()", func() {
+	DescribeTable("ValidateClaimValidationRules() test case", func(input interface{}, shouldErr bool) {
+		err := ValidateClaimValidationRules(input)
+		if shouldErr {
+			Expect(err).To(HaveOccurred())
```

Comment:
@chenz4027 Would you be able to extend the test here to check that it's getting the expected error. Probably the easiest way would be to check that the error message is starting with a certain substring I think.

Reply:
Will add more tests about the actual failures

Reply:
Added the actual failure, also added more tests for IsExternalAuthAvailable